From e9a260ad97fa92605397fca5d2a1a59b43dba3ba Mon Sep 17 00:00:00 2001
From: Fabrice Gasnier <fabrice.gasnier@foss.st.com>
Date: Mon, 29 Jan 2024 16:25:58 +0100
Subject: [PATCH 0634/1695] dts: activate lptimer3 for stm32mp135 disco boards

Activate lptimer3 on stm32mp135 disco boards, that can be used to monitor
HSE clock

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git bf8ebc06ee6f30374df4ca79efd038b88b164112 dts: activate lptimer3 for stm32mp135 disco boards]
Change-Id: I4e1fda976a245c3140fe2b4ff6185e0027acce74
Signed-off-by: Ludovic Barre <ludovic.barre@foss.st.com>
Signed-off-by: Fabrice Gasnier <fabrice.gasnier@foss.st.com>
---
 core/arch/arm/dts/stm32mp135f-dk.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp135f-dk.dts b/core/arch/arm/dts/stm32mp135f-dk.dts
index 32d7c6ab4..4f5917ce5 100644
--- a/core/arch/arm/dts/stm32mp135f-dk.dts
+++ b/core/arch/arm/dts/stm32mp135f-dk.dts
@@ -352,6 +352,14 @@
 	status = "okay";
 };
 
+&lptimer3 {
+	status = "okay";
+
+	counter {
+		status = "okay";
+	};
+};
+
 &oem_enc_key {
 	st,non-secure-otp-provisioning;
 };
-- 
2.47.2

